test/distributed/test_distributed_fork.py

```diff
@@ -11,6 +11,8 @@
     DistributedTest, TestDistBackend
 )
 
+torch.backends.cuda.matmul.allow_tf32 = False
```

Comment:
I am not familiar with the impact of this configuration, but this looks in general OK to me and tests suggests this is fine at least in our CI. 

@ngimel please correct me if I was wrong. Thanks!